backend/routes/scoreCalculations.js

```diff
@@ -0,0 +1,29 @@
+import { PrismaClient } from "@prisma/client";
+const prisma = new PrismaClient();
+
+async function calcPerformanceScores(playerId) {
+  let player = await prisma.player.findUnique({
+    where: {
+      id: playerId,
+    },
+  });
+  let outsideOffenseScore = 40;
+
+  if (player.field_attempts !== 0) {
+    outsideOffenseScore =
+      (player.three_attempts / player.field_attempts) * 50 +
+      (player.three_percent - 0.33) * 1000;
+  }
```

Comment:
+a to Mike's comment. Rather than having an API route for this, couldn't this be calculated when the `Player` itself is created and then stored on the Player itself rather than on the team or opponent player?

Reply:
Initially I thought no, but I'm now starting to think yes. The two reasons that I wanted each team type to have it's own table are that 1) a truly robust algorithm would take teammates into account. So, the same player on two different teams should have two different scores. I wouldn't be adding that algorithm right now, because of time, but I wanted to keep it an option for scalability 2) If I could keep track of every user's players, I wouldn't have to make unnecessary calculations for players that don't belong on any team. Yi also brought up that point, and I'm realizing the flaws in both of my arguments, so I'm going to be changing that.